cmd/sync/sync.go

```diff
@@ -68,6 +69,13 @@ func (app *SyncApp) Cleanup() {
 
 }
 
+type mockTxProcessor struct {
+}
+
+func (mockTxProcessor) ValidateTransactionSignature(tx *types.SerializableSignedTransaction) (address.Address, error) {
+	return address.HexToAddress("0xFFFF"), nil
```

Comment:
why mock?

Reply:
@almogdepaz we actually need the TransactionProcessor to remove this mock. I didn't want to change it without talking to you since the TP is not created in your cmd/sync app.
Should I create and use it?

Reply:
leave the mock! this tests only the sync  the test isolation is on purpose
we should have integration tests with other modules but this is more basic